docs/pages/components/index.tsx

```diff
@@ -53,7 +53,7 @@ export default function PackagesHome({
 
 				return (
 					p.title.toLowerCase().includes(searchTerm.toLowerCase()) ||
-					p.groupName.toLowerCase().includes(searchTerm.toLowerCase()) ||
+					// p.groupName.toLowerCase().includes(searchTerm.toLowerCase()) ||
 					p.description?.toLowerCase().includes(searchTerm.toLowerCase())
```

Comment:
@stowball I need help figuring out how to apply this fuzzy search logic to the side nav. I don't know how to get the description into the component page side nav list data.